invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/deposit.html

```diff
@@ -27,6 +27,10 @@
   {%- if permissions %}
   <input id="deposits-record-permissions" type="hidden" name="deposits-record-permissions" value='{{permissions | tojson }}'></input>
   {%- endif %}
+  {%- if sentry_config %}
```

Comment:
This might be common also with search at some point. Creating a common base file and adding this block there would be an option.